index.html

```diff
@@ -34,233 +38,314 @@ <h1 class="name">
 					Start collaboration
 				</button>
 			</div>
+			<div class="image_dots"></div>
 			<div class="bottom_geometry"></div>
 		</section>
 
 		<section class="project_section">
 			<h2 class="projects_label">
 				Projects
 			</h2>
-			<div class="project_card">
-				<div class="project_image"></div>
+			<div class="image_dot_1"></div>
+			<div class="project_card_macro">
```

Comment:
- [ ] Impressive animation! Now let's align these popup language tags vertically centered like in the design. 💪 

![image](https://user-images.githubusercontent.com/57421823/149366319-b1216a93-a7b1-45d5-81f3-bcbfe1017180.png)